.platform.app.yaml

```diff
@@ -21,12 +21,13 @@ mounts:
 hooks:
   build: |
     set -e
+    OPENSWOOLE_VERSION=22.0.0
     echo "Installing Swoole extension"
-    bash install-swoole.sh openswoole 22.0.0
+    bash install-swoole.sh openswoole "$OPENSWOOLE_VERSION"
     echo "Current PHP modules:"
     php -m
     echo "Installing platform.sh CLI"
-    curl -sS https://platform.sh/cli/installer | php
+    curl -fsSL https://raw.githubusercontent.com/platformsh/cli/main/installer.sh | bash
```

Comment:
Wouldn't it be better to use a version number instead of the branch name here? Would one be available?

Reply:
That is how platfom.sh suggests their cli should be installed and I don't care tbh.
Installer uses latest version so tagging does not matter and it is only used to trigger re-deployment from cron once in a blue moon to refresh certs.